src/cookies.njk

```diff
@@ -0,0 +1,75 @@
+---
+title: Cookies
+description: GOV.UK Design System puts small files (known as ‘cookies’) on your computer
+layout: layout-single-page.njk
+---
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set successNotificationBannerHtml %}
+  <p class="govuk-notification-banner__heading">
+    You’ve set your cookie preferences. <a class="govuk-notification-banner__link" href="#">Go back to the page you were looking at</a>.
+  </p>
+{% endset %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="app-cookies-page" data-module="app-cookies-page">
+      <div class="js-cookies-page-error-summary" hidden>
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: [
+            {
+              text: "Select yes to accept analytics cookies",
+              href: "#analytics"
+            }
+          ]
+        }) }}
+      </div>
+
+      <div class="js-cookies-page-success" hidden>
+        {{ govukNotificationBanner({
+          type: "success",
+          html: successNotificationBannerHtml
+        }) }}
+      </div>
+
+      <h1 class="govuk-heading-xl">Cookies</h1>
+      <p class="govuk-body"><a href="/" class="govuk-link">GOV.UK Design System</a> puts small files (known as ‘cookies’) on your computer.</p>
+
+      <h2 class="govuk-heading-l">Essential cookies (strictly necessary)</h2>
+      <h2 class="govuk-heading-l">Analytics cookies (optional)</h2>
+
+      <div class="js-cookies-page-form-group" hidden>
```

Comment:
I tripped over the naming of this a few times whilst reviewing, as the radios are already wrapped in a 'form group' within the component.

For example, the show/hide error message functions use a query selector to find the `form-group--error` class, but because both things are named form groups I was confused as I was expecting the `--error` modifier class to be on the `$cookieFormGroup` itself, rather than a child.

Reply:
I'd suggest moving this class onto the form group – which I think would allow the error message code to be simplified quite a lot, but unfortunately there's no way to add the `hidden` attribute to the form group using the macro.

Reply:
@36degrees For now, I've just [renamed the class](https://github.com/alphagov/govuk-design-system/pull/1745/commits/911d5c9c4d5c67793899de522bcdfe0d7a8fbb54#diff-b65e99e4a965a6c092b03d9b6554eed480dce2d21658b588afd40e248f2a7e8fR51) (and also made it a form element in a [later commit](https://github.com/alphagov/govuk-design-system/pull/1745/commits/ce91eb12fee3ecc65bf0c0333b6d4c6f35389ab6))

Do you know if there's an existing card anywhere to support attributes on the formGroup? Happy to write one up if not.